tests/integration/delete.bats

```diff
@@ -51,3 +51,36 @@ function teardown() {
   runc delete --force notexists
   [ "$status" -eq 0 ]
 }
+
+@test "runc delete --force in cgroupv2 with subcgroups" {
+  requires cgroups_v2 root
+  set_cgroups_path "$BUSYBOX_BUNDLE"
+
+  # grant `rw` priviledge to `/sys/fs/cgroup`
+  cat ${BUSYBOX_BUNDLE}/config.json\
+   | jq '.mounts |= map((select(.type=="cgroup") | .options -= ["ro"] | .options += ["rw"]) //.)'\
+   > ${BUSYBOX_BUNDLE}/config.json
```

Comment:
Why `>` can't work in here?

Reply:
It can work in my pc:
```
[root@localhost runc]# make localintegration TESTPATH=/delete.bats RUNC_USE_SYSTEMD=1
go build "-mod=vendor" -buildmode=pie  -tags "seccomp selinux apparmor" -ldflags "-X main.gitCommit="98a52396ea5807c141118704e0540b76edc5c03e" -X main.version=1.0.0-rc10+dev " -o runc .
go build "-mod=vendor" -buildmode=pie  -tags "seccomp selinux apparmor" -ldflags "-X main.gitCommit="98a52396ea5807c141118704e0540b76edc5c03e" -X main.version=1.0.0-rc10+dev " -o contrib/cmd/recvtty/recvtty ./contrib/cmd/recvtty
bats -t tests/integration/delete.bats
1..4
ok 1 runc delete
ok 2 runc delete --force
ok 3 runc delete --force ignore not exist
ok 4 runc delete --force in cgroupv2 with subcgroups
[root@localhost runc]#
```